### PR TITLE
Multiple assignment case

### LINKED ISSUE #22: typedarray-pool is broken
Stumbled upon this in [tinyify](https://ghub.io/tinyify) not working in [dy/budo](https://github.com/dy/budo). [typedarray-pool](https://ghub.io/typedarray-pool) turned out to not shake properly.

The bug case:
```js
// a.js
exports.x0 =
exports.x1 =
exports.x2 = function () {}
```
```js
// b.js
require('./a').x0()
```

Produces
```js
(function(){function r(e,n,t){function o(i,f){if(!n[i]){if(!e[i]){var c="function"==typeof require&&require;if(!f&&c)return c(i,!0);if(u)return u(i,!0);var a=new Error("Cannot find module '"+i+"'");throw a.code="MODULE_NOT_FOUND",a}var p=n[i]={exports:{}};e[i][0].call(p.exports,function(r){var n=e[i][1][r];return o(n||r)},p,p.exports,r,e,n,t)}return n[i].exports}for(var u="function"==typeof require&&require,i=0;i<t.length;i++)o(t[i]);return o}return r})()({1:[function(require,module,exports){
exports.x0 =
/* common-shake removed: exports.x1 = */ void 0, /* common-shake removed: exports.x2 = */ function () {}

},{}],2:[function(require,module,exports){
require('./a').x0()

},{"./a":1}]},{},[2]);
```

`exports.x0` here === `void 0`

@goto-bus-stop

### PR DESCRIPTION
**Pull request by dy**
Fixes #22.
Tested out with huge bundle in [audio-formant](https://github.com/formant/audio-formant/blob/master/test.js), seems to work ok now.
All tests pass, not sure what other edge cases there can be.
@goto-bus-stop

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -186,7 +186,14 @@ function createStream (opts) {
           node.right.type === 'AssignmentExpression' ||
           // Don't output a statement containing only `void () => {}`
           node.right.type === 'ArrowFunctionExpression') {
-        prefix += 'void 0, '
+        // ignore alias assignment expression `exports.a = exports.b = exports.c`
+        if (!(
+          node.right.type === 'AssignmentExpression' &&
```
Comment (dy) [inline index.js]: The naive guess, possibly can be more general.

File: test/funny-exports/expected.js [hunk 2]
```diff
@@ -11,6 +11,6 @@ console.log(/* common-shake removed: exports.a = */ 'hello world')
 function a() {} /* common-shake removed: exports.b = */ void 0, function () { return a }, console.log(a)
 var c = (0, /* common-shake removed: exports.c = */ 'beep boop')
 
-/* common-shake removed: exports.d = */ void 0, exports.e = null
```
Comment (dy) [inline test/funny-exports/expected.js]: Bonus side effect

File: test/test.js [hunk 3]
```diff
@@ -25,8 +25,8 @@ function runTest (t, name) {
 
   bundle.pipe(concat(function (result) {
     t.is(
-      result.toString('utf8'),
-      fs.readFileSync(expected, 'utf8'),
+      result.toString('utf8').replace(/\r\n/g, '\n'),
```
Comment (dy) [inline test/test.js]: Windows newlines are different

**Comment by goto-bus-stop** [comment]
:package: 0.5.3
